README.md

```diff
@@ -86,11 +88,11 @@ done
 
 At this point, multicluster-scheduler is installed in both clusters, but neither is a source or target yet.
 
-1. Target clusters need service accounts and RBAC rules to autenticate and authorize source clusters, cf. [Creating Service Accounts and RBAC Rules in Target Clusters](#1-creating-service-accounts-and-rbac-rules-in-target-clusters), below.
+1. Target clusters need service accounts and RBAC rules to autenticate and authorize source clusters. The source controller can configure those for you, cf. [Creating ClusterSources and Sources in Target Clusters](#1-creating-clustersources-and-sources-in-target-clusters), below.
```

Comment:
"The source controller can ..." is confusing; seems to imply that you do this in the source cluster (while in reality it is done in the target cluster).

I would recommend expanding it to "The source controller in  the target cluster can ..."
I know it is a bit more verbose (and repetitive), but would be much easier to understand for a novice user, in my opinion.

Reply:
:+1: